Make sure the dependency exists, as testfiletemplates depends on ui files
generated by kdevfiletemplates.

Submitted upstream: https://phabricator.kde.org/D1160
--- plugins/filetemplates/CMakeLists.txt.orig	2016-03-17 11:59:47 UTC
+++ plugins/filetemplates/CMakeLists.txt
@@ -81,6 +81,7 @@ main.cpp
 )
 
 kde4_add_executable(testfiletemplates ${test_srcs})
+add_dependencies(testfiletemplates kdevfiletemplates)
 
 target_link_libraries(testfiletemplates
     ${KDE4_KDECORE_LIBS}
